common/tools/dev-tool/src/util/samples/generation.ts

```diff
@@ -115,7 +115,12 @@ export async function makeSampleGenerationInfo(
     }
   };
 
-  const moduleInfos = await processSources(sampleSourcesPath, sampleSources, fail, requireInScope);
+  const moduleInfos = await processSources(
+    sampleSourcesPath,
+    sampleSources.sort(),
```

Comment:
so this is the only real change of adding `.sort()`.  All others are formatting changes.

Reply:
Thanks